lib/packages.cf

```diff
@@ -71,6 +71,9 @@ bundle common package_module_knowledge
 
     slackware::
       "platform_default" string => "slackpkg";
+
+    windows.(cfengine_3_12|cfengine_3_13|cfengine_3_14|cfengine_3_15).!cfengine_3_12_0::
```

Comment:
Shouldn't this one exclude 3.12.0 and 3.12.1?

like the guard used on inputs?
     windows.cfengine_3_12.!(cfengine_3_12_0|cfengine_3_12_1)::

Reply:
thanks, I missed this